cuvs-bench/cpu/Dockerfile

```diff
@@ -55,8 +55,7 @@ CMD ["--dataset fashion-mnist-784-euclidean", "", "--algorithms hnswlib"]
 
 ENTRYPOINT ["/bin/bash", "/data/scripts/run_benchmark.sh"]
 
-
-FROM bench-base AS cuvs-bench-cpu-datasets
+FROM cuvs-bench-cpu AS cuvs-bench-cpu-datasets
```

Comment:
This is a serious bug in this Dockerfile!

And `hadolint` caught it (indirectly) 😁 

```text
DL3006 warning: Always tag the version of an image explicitly
```

There's not an earlier target called `bench-base`, so trying to build this would try to pull an image from DockerHub called `bench-base`. I'm pretty confident this is not what we want.